### PR TITLE
Include a minimal mix.exs build file

### PR DESCRIPTION
**Pull request by michaelklishin**
So that this project can be compiled by Mix as a dependency in an Elixir
code base (in my case, RabbitMQ CLI tools).

Pair: @dumbbell.

### CODE REVIEW

**Comment by michaelklishin** [comment]
@zhongwencool @dumbbell and I did a bit of `.app.src` massaging that would have to go with this change. I hope you don't mind it, we didn't mean to remove you as a maintainer ;) but Rebar3 asked for the field to be removed.

File: mix.exs [hunk 1]
```diff
@@ -0,0 +1,17 @@
+defmodule ObserverCli.MixProject do
+  use Mix.Project
+
+  def project do
+    [
+      app: :observer_cli,
+      version: "0.4.4",
```
[the remark this answers is not in the document]
Comment (michaelklishin) [reply mix.exs]: Oops. Done.

**Comment by michaelklishin** [comment]
@zhongwencool thank you. Can we get a new tag, please? 🙏 (no need for a Hex release if you don't feel like it)

**Comment by zhongwencool** [comment]
@michaelklishin done :0

**Comment by dumbbell** [comment]
Hi!

We do need a release on Hex.pm however. Our dependencies are taken from there and the build tool is also documented there.

Thank you for your quick feedback so far! Much appreciated!

**Comment by zhongwencool** [comment]
@dumbbell 1.4.5

**Pull request merged**